app/assets/stylesheets/rubrics.scss

```diff
@@ -27,7 +27,7 @@
 }
 
 #rubric_levels_pane {
-  @include border-radius($radius);
+  border-radius: $radius;
```

Comment:
Properties should be ordered border, border-radius, display, margin-left, padding, width